ern-api-impl-gen/src/generators/ApiImplGen.ts

```diff
@@ -63,9 +63,7 @@ export default class ApiImplGen {
               apiPackagePath,
               PackagePath.fromString(`react-native@${reactNativeVersion}`),
               PackagePath.fromString(
-                `react-native-electrode-bridge@${
-                  reactNativeElectrodeBridge.version
-                }`
+                `react-native-electrode-bridge@${reactNativeElectrodeBridge.version}`
```

Comment:
Looks like a formatting-only change.. Just friendly request to use a separate commit next time for easier review ;)

Reply:
Totally agreed and would make the same remark to anyone ;)

Not my intention though, it slipped through it seems ... looks like `prettier` was applied on this file as I have enabled `format on save` in vs-code. Even if I had not enabled it, it would have updated the file via a pre-commit hook (which is running `prettier` on all staged files, in case contributors are not using vs-code or format on save).

That being said, I had already noticed this behavior but never looked into it more closely ... just took a closer look and looks like we should run prettier on all source files whenever we are bumping the version of it, has it comes with some formatting rules updates. I think this is what happened here and in the past when I noticed this behavior. Because `prettier` is only run on any saved files (with proper vs-code config) or any git staged files as pre-commit hook ... if we bump prettier version and it comes with new formatting rules, then this new formatting rules will be only applied when the file will be touched (either as save or git staged). 

So rule of thumb, we should run prettier on all source files whenever we bump the version of it (should be done in same PR as the one containing bump to prettier so that we can see all files for which formatting was changed by new version of it).

For now, I'll just run `prettier` on all files and open a PR with changed files. This way at least this kind of issue won't happen on any future PR, at least until next version bump of prettier (then we have to keep in mind that we should re-run prettier post version bump of it... and can think in // of a way to automate this so that we don't have to keep it in mind at all ;P)

Reply:
Sounds good, and I agree - Updates to the formatter that come with change to how files are formatted need to go in the _same commit_. Otherwise, there is no other way to avoid inconsistent state. I have a feeling that a different version of Prettier also caused the unintended formatting changes in #1441 a few weeks ago. Overall, this project is already in a pretty good state (compared to many others), but we can make it even better. Let's chat next week 👍